File: kizizip/01/[PGS - Lv1] 서울에서 김서방 찾기.java
class Solution {
    public String solution(String[] seoul) {
        String answer = "";
        
        int index = 0;
        for (int i=0; i<seoul.length; i++) {
            if (seoul[i].equals("Kim")) {
                index = i;
                break;
            }
        }
        
        answer = "김서방은 " + index + "에 있다";
        
        return answer;
    }
}

File: 7월 4주차/JeongEon8/01/[PGS - Lv1] 서울에서 김서방 찾기.cpp
#include <string>
#include <vector>
#include <string>

using namespace std;

string solution(vector<string> seoul) {
    string answer = "김서방은 ";
    
    int cnt = seoul.size();
    for(int i = 0; i < cnt; i++){
        if(seoul[i] == "Kim"){
            answer += to_string(i);
            break;
        }
    }
    
    answer += "에 있다";
    
    return answer;
}
